sbom_formatter.go

```diff
@@ -0,0 +1,26 @@
+package packit
+
+import "io"
+
+// SBOMFormat represents the mapping of a formatted SBOM content to its file
+// extension on the filesystem.
+type SBOMFormat struct {
+	Extension string
+	Content   io.Reader
+}
+
+// SBOMFormatter defines the interface for
```

Comment:
Are you missing the back half of a sentence here?

Reply:
LOL